conda_build/develop.py

```diff
@@ -180,3 +180,106 @@ def execute(
         # they use the conda environment for loading libraries at runtime
         relink_sharedobjects(pkg_path, prefix)
         print("completed operation for: " + pkg_path)
+
+
+def execute(
+    recipe_dirs: list[str],
+    prefix: str = sys.prefix,
+    no_pth_file: bool = False,
+    build_ext: bool = False,
+    clean: bool = False,
+    uninstall: bool = False,
+) -> None:
+    """
+    Use pypa/build to develop.
+    """
+    if not isdir(prefix):
+        sys.exit(
+            f"""\
+Error: environment does not exist: {prefix}
+#
+# Use 'conda create' to create the environment first.
+#"""
+        )
+
+    python_executable = find_executable("python", prefix=prefix)
+    assert python_executable
+
+    # current environment's site-packages directory
+    sp_dir = get_site_packages(
+        prefix, ".".join((str(sys.version_info.major), str(sys.version_info.minor)))
+    )
+
+    if isinstance(recipe_dirs, str):
+        recipe_dirs = [recipe_dirs]
+
+    for path in recipe_dirs:
+        pkg_path = abspath(expanduser(path))
+
+        if uninstall:
+            raise NotImplementedError("nope")
+            # uninstall then exit - does not do any other operations
```

Comment:
Ask them to use conda uninstall or pip uninstall since these are installed like standard Python packages

Reply:
Could we convert the ephemeral wheel into a conda package and install that instead? Or would it be too much trouble to avoid keeping that in conda's package cache?